packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -243,64 +267,53 @@ export default function AccessRequirementList(
           File(s)
         </>
       )
-    return (
-      <>
-        <IconSvg icon="file" sx={{ width: '30px' }} />{' '}
-        <Link
-          href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!Synapse:${entityId}`}
-        >
-          {entityInformation[0]?.name}
-        </Link>
-      </>
-    )
-  }, [entityId, entityInformation, numberOfFilesAffected, requestObjectName])
+    if (subjectType === RestrictableObjectType.ENTITY) {
+      return <EntityLink entity={subjectId} />
+    } else if (subjectType === RestrictableObjectType.TEAM) {
+      return <UserOrTeamBadge principalId={subjectId} />
+    } else {
+      console.warn(
+        `Unhandled case for AccessRequirementList requestDetails: ${subjectType}: ${subjectId}`,
+      )
+      return <></>
```

Comment:
At the moment, the only remaining case in [RestrictableObjectType](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/RestrictableObjectType.html) is EVALUATION--I've never heard of an AR on an evaluation (since it's usually applied to the participant team. Do we need to address this ASAP?

Reply:
Please ignore the EVALUATION subject type for now (no known use case)